datasync/chaindatafetcher/kas/repository.go

```diff
@@ -44,13 +50,16 @@ const (
 	connMaxLifetime   = 24 * time.Hour
 	maxDBRetryCount   = 20
 	DBRetryInterval   = 1 * time.Second
+	cacheTimeout      = 200 * time.Millisecond
```

Comment:
How about this?
```suggestion
	apiCtxTimeout      = 200 * time.Millisecond
```

Reply:
Applied. 220e26ebbc270aa1ec184b3f5e07589bad9ead46